cms/templates/index.html

```diff
@@ -227,10 +227,11 @@ <h3 class="course-title">${course_info['display_name'] | h}</h3>
                 </div>
 
                 <dl class="course-status">
-                  <dt class="label sr">This re-run processing status:</dt>
+                  ## Translators: This is a status message for the course re-runs feature. When a course admin indicates that a course should be re-run, the system needs to process that and get the new course re-run ready to go. The status of that re-run process will follow this text.
```

Comment:
Updated. I thought that I was unable to have translators comment span more than one line, because every other translators comment in this file is only one line. I added a new commit to wrap all the translators comments, so that it's less confusing for people writing translators comments in the future.